### ISSUE TITLE
Override disableForSelector

### ISSUE THREAD

**Issue by jayzawrotny** [atom/autocomplete-snippets#63]
Is there a way to override the disableForSelector in a general autocomplete option or a local config setting? I would very much like to use some jsdoc autocomplete snippets within a comment block and that disableForSelector seems to be explicitly disabling it from being listed.

**Comment by lee-dohm** [comment]
What `disableForSelector` are you referring to? The autocomplete-snippets package doesn't have a setting called this.

**Comment by jayzawrotny** [comment]
https://github.com/atom/autocomplete-snippets/blob/master/lib/snippets-provider.coffee#L4

> ```js
class SnippetsProvider
  selector: '*'
  disableForSelector: '.comment, .string'
```

**Comment by tccolestock** [comment]
Since this is still open I suppose I'll leave my comment / request here. I would also like the ability (or information on how) to override the disableForSelector option @jayzawrotny mentioned. When I'm using LaTeX, the math blocks are under the scope .string which prevents me from seeing the snippet suggestion box.  

I have read a lot of posts discussing this same issue, but none seem to have come to a definitive answer / solution on how to resolve this. The ability to use snippets with LaTeX will be very convenient, but only being able to do so in text mode, is well, kind of useless (at least for me). I tried to edit the `config.cson` file to override the disableForSelector option with: 
```
  "autocomplete-snippets": 
       disableForSelector: ".comment"
```
But that did not have any effect. Any idea on how to implement this?

**Comment by lee-dohm** [comment]
The autocomplete-snippets package doesn't currently expose this value for configuration. A code change would need to be made to expose it.

**Comment by tccolestock** [comment]
@lee-dohm thank you for the reply. That certainly explains my attempt not having any effect. Would such a code change be possible? Even if everything still defaults to how it currently is, then the users that would like to add snippets to comments or strings can set that in the `config.cson` file.

**Comment by lee-dohm** [comment]
> Would such a code change be possible?

I'm pretty certain it is. (That's why I added the `enhancement` label :grinning:) I can't commit to any ETA though 😞

**Comment by freistil90** [comment]
Leaving my comment here as well, since is the currently only big issue I have with atom. Great work otherwise.
Is it really that much work to change it?

**Comment by Aerijo** [comment]
@lee-dohm This issue was (somewhat) resolved in 037953b. It's not configurable, but it seems everyone here just wanted it removed anyway. Should this be closed?

**Comment by lee-dohm** [comment]
Thanks @Aerijo. Yes, I do believe it should be :+1:
